File: app/src/main/java/com/vesystem/ui/interfaces/IOperatorMenu.java
package com.vesystem.ui.interfaces;

import android.view.View;

/**
 * Created Date 2020-01-13.
 *
 * @author ChenRui
 * ClassDescription：
 */
public interface IOperatorMenu {

    //触屏点击
    void touchClick(View view);

    //触摸移动
    void touchMove(View view);

    //拖拽选择
    void dragSelect(View view);

}

File: spice/src/main/java/com/vesystem/spice/ui/interfaces/IOperatorMenu.kt
package com.vesystem.spice.ui.interfaces

import android.view.View

/**
 * Created Date 2020-01-13.
 *
 * @author ChenRui
 * ClassDescription：
 */
interface IOperatorMenu {
    //触屏点击
    fun touchClick(view: View)

    //触摸移动
    fun touchMove(view: View)

    //拖拽选择
    fun dragSelect(view: View)
}